projects/plugins/social/src/class-note.php

```diff
@@ -31,6 +31,7 @@ public function init() {
 		}
 		static::register_cpt();
 		add_action( 'wp_insert_post_data', array( new Note(), 'set_empty_title' ), 10, 2 );
+		add_filter( 'the_title', array( new Note(), 'override_empty_title' ), 10, 2 );
```

Comment:
Perhaps we could reverse this logic and only add this filter if we are on the correct screen. We could move this to the `admin_init` hook too, to make sure it's only run in the admin area

Reply:
I have done this, let me know what you think